sktime/forecasting/enbpi.py

```diff
@@ -128,13 +134,41 @@ def __init__(
 
         super().__init__()
 
-        from tsbootstrap import MovingBlockBootstrap
+        from tsbootstrap import BaseTimeSeriesBootstrap, MovingBlockBootstrap
 
-        self.bootstrap_transformer_ = (
-            clone(bootstrap_transformer)
-            if bootstrap_transformer is not None
-            else MovingBlockBootstrap()
-        )
+        if isinstance(bootstrap_transformer, BaseTimeSeriesBootstrap):
```

Comment:
this inheritance check and import can be avoided by doing the type check, using a check for the tag `object_type == "bootstrap"`